src/common/intl.cpp

```diff
@@ -1898,7 +1898,7 @@ wxString GetDateFormatFromLangInfo(wxLocaleInfo index)
 wxString wxLocale::GetInfo(wxLocaleInfo index, wxLocaleCategory cat)
 {
 // TODO: as of 2014 Android doesn't has complete locale support (use java api)
-#if !(defined(__WXQT__) && defined(__ANDROID__))
+// DONE: Supported by NDK21d
```

Comment:
Let's just remove _both_ of these lines.

Reply:
Ok